packages/mds-daily/tools/metrics-sheet/geo-metrics.js

```diff
@@ -0,0 +1,96 @@
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+require('dotenv').config()
+const GoogleSpreadsheet = require('google-spreadsheet')
+const { promisify } = require('util')
+const rp = require('request-promise')
+
+// The list of providers ids on which to report. TODO: get from config store at some point
+const reportProviders = [
+  'c20e08cf-8488-46a6-a66c-5d8fb827f7e0', // JUMP
```

Comment:
We really should create constants for these UUIDs in `mds-providers`

Reply:
The `package-lock.json` file should be removed.

Reply:
Any plans to use Typescript here?

Reply:
> We really should create constants for these UUIDs in `mds-providers`

I have a separate PR that defines said constants.

Reply:
Agree on the constants.  However, in this case I don't want all providers, only those that are "active" (e.g. not Clevr, Audit, etc).  Belongs in a config of some kind.

`package-lock.json` removed.

Would like to convert to typescript, but requires I do some learning.  You are welcome to take a shot.

Reply:
@whereissean Understand you want only active providers, but with my PR you will be able to declare them using constants, like as follows:
```
// The list of providers ids on which to report
const reportProviders = [
  JUMP_PROVIDER_ID,
  LIME_PROVIDER_ID,
  BIRD_PROVIDER_ID,
  LYFT_PROVIDER_ID,
  WHEELS_PROVIDER_ID,
  SPIN_PROVIDER_ID,
  SHERPA_PROVIDER_ID
]
```